src/components/organize/journeys/JourneyPerson.tsx

```diff
@@ -0,0 +1,36 @@
+import { Box } from '@material-ui/core';
+import { Close } from '@material-ui/icons';
+import { grey } from '@material-ui/core/colors';
+import { useState } from 'react';
+
+import ZetkinPerson from 'components/ZetkinPerson';
+import { ZetkinPerson as ZetkinPersonType } from 'types/zetkin';
+
+const JourneyPerson = ({
+  person,
+  onRemove,
+}: {
+  onRemove: (person: ZetkinPersonType) => void;
+  person: ZetkinPersonType;
+}): JSX.Element => {
+  const [hover, setHover] = useState<boolean>(false);
+  return (
+    <Box
+      alignItems="center"
+      bgcolor={hover ? grey[200] : ''}
+      display="flex"
+      justifyContent="space-between"
+      onMouseEnter={() => setHover(true)}
+      onMouseLeave={() => setHover(false)}
+      p={1}
+    >
+      <ZetkinPerson
+        id={person.id}
+        name={`${person.first_name} ${person.last_name}`}
+      />
+      {hover && <Close color="secondary" onClick={() => onRemove(person)} />}
```

Comment:
I think it would be nice with a `pointer` cursor on this close button. I missed it when I was trying it out.

Reply:
Agree!